hasjob/static/js/app.js

```diff
@@ -184,18 +185,19 @@ window.Hasjob.Filters = {
         selectedTypes: window.Hasjob.Config.selectedTypes,
         selectedCategories: window.Hasjob.Config.selectedCategories,
         selectedQuery: window.Hasjob.Config.selectedQuery,
-        selectedEquity: window.Hasjob.Config.selectedEquity
+        selectedEquity: window.Hasjob.Config.selectedEquity,
+        filtersMenuOpen: false
```

Comment:
Does this variable signify that the filter menu has been opened as a sidebar? Maybe we could just use `sidebarOn`?